docs/application/tizen-studio/native-tools/ui-builder/custom-ui-component.md

```diff
@@ -55,17 +55,17 @@ A custom component requires an EDJ resource file (`[ComponentName].edj`) and an
-    | <state>     | <value>     | signal        | EDC  signal used to change the UI state at runtime.  For information on using signals, see [Program Block](../../../org.tizen.guides/html/native/ui/efl/learn_edc_program_n.htm). | String  | Yes      | -                                    |
+    | <state>     | <value>     | signal        | EDC  signal used to change the UI state at runtime.  For information on using signals, see [Program Block](../native/guides/ui/efl/learn-edc-program.md). | String  | Yes      | -                                    |
     | <state>     | <value>     | source        | User-defined  parameter sent when the EDC signal is triggered | String  | No       | null                                 |
     | <component> | <text>      | name          | Text  property identifier                | String  | Yes      | -                                    |
     | <component> | <text>      | display_name  | Name  shown on the text property label in the Properties view | String  | No       | name attribute in the <text>element  |
     | <component> | <text>      | description   | Text  property description               | String  | No       | null                                 |
-    | <component> | <text>      | part_name     | part block to change when the text property is modified.  For more information, see [Part Block](../../../org.tizen.guides/html/native/ui/efl/learn_edc_part_n.htm). | String  | Yes      | -                                    |
+    | <component> | <text>      | part_name     | part block to change when the text property is modified.  For more information, see [Part Block](../native/guides/ui/efl/learn-edc-part.md). | String  | Yes      | -                                    |
     | <component> | <text>      | default_value | Text property  default value             | String  | No       | null                                 |
     | <component> | <color>     | name          | Color  property identifier               | String  | Yes      | -                                    |
     | <component> | <color>     | display_name  | Name  shown on the color property label in the Properties view | String  | No       | name attribute in the <color>element |
     | <component> | <color>     | description   | Color  property description              | String  | No       | null                                 |
-    | <component> | <color>     | class_name    | EDC  color class name.  For more information, see [Color Classes Block](../../../org.tizen.guides/html/native/ui/efl/learn_edc_color_classes_n.htm). | String  | Yes      | -                                    |
+    | <component> | <color>     | class_name    | EDC  color class name.  For more information, see [Color Classes Block](../native/guides/ui/efl/learn-edc-color-classes.md). | String  | Yes      | -                                    |
```

Comment:
It looks `../../../native/guides/ui/efl/learn-edc-color-classes.md` is correct.

Reply:
done